public/js/views.js

```diff
@@ -933,7 +933,30 @@ App.Views.CommentsList = Backbone.View.extend({
         this.collection.on('remove', this.render, this);
         this.collection.on('add', this.renderLastComment, this);
         Backbone.Validation.bind(this);
-        App.poller = Backbone.Poller.get(this.collection, {delay: 2000}).start();
+        //App.poller = Backbone.Poller.get(this.collection, {delay: 2000});
+
+        //Web socket subscriber
+        ab.connect(
+            'ws://' + window.location.hostname + ':8080',
```

Comment:
Invalid location and port:
window.location.hostname produces path without prefix
port should be different than 8080 and assigned in config